react/features/analytics/actionTypes.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * The type of (redux) action to adjust the analytics config.
+ *
+ * {
+ *     type: SET_ANALYTICS_CONFIG,
+ *     available: boolean
+ * }
+ */
+export const SET_ANALYTICS_CONFIG = Symbol('SET_ANALYTICS_CONFIG');
```

Comment:
I think we can implement what we need without the extra action and state. In essence, this action duplicates state already present in features/base/config, so I'd rather use that.

If we feel the need to have a succinct way to know if analytics are enabled, we can have a function which consults the state and returns a boolean.